### PR TITLE
update access point management

### PR DESCRIPTION
**Pull request by mbeh**
* set default format for access rules to ranges in access configuration overview table
* fix not showing ip ranges when editing an access configuration
* renamed new/Neu button to add/Hinzufügen in ip edit form
* add new access configuration types
* link platforms and access configurations

### CODE REVIEW

File: app/grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -8,6 +8,9 @@ import org.apache.log4j.RollingFileAppender
 
 grails.project.groupId  = appName // change this to alter the default package name and Maven publishing destination
 grails.config.locations = ["file:${userHome}/.grails/${appName}-config.groovy"]
+grails.config.locations = ["file:/etc/tomcat/conf.d/${appName}-config.groovy"]
```
Comment (croqueGrec09) [inline app/grails-app/conf/Config.groovy]: three times the same config? Call at Freiburg!

File: app/grails-app/views/organisation/accessPoints.gsp [hunk 2]
```diff
@@ -10,7 +10,18 @@
...
-    <g:render template="breadcrumb" model="${[ orgInstance:orgInstance, params:params ]}"/>
+
+    <semui:breadcrumbs>
+        <semui:crumb controller="organisation" action="show" id="${orgInstance.id}" text="${orgInstance.getDesignation()}" />
+        <semui:crumb message="org.nav.accessPoints" class="active"/>
+    </semui:breadcrumbs>
+
+    <g:if test="${accessService.checkPerm('ORG_INST,ORG_CONSORTIUM')}">
+        <semui:controlButtons>
+            <g:render template="actions" model="[org:org]" />
```
Comment (croqueGrec09) [inline app/grails-app/views/organisation/accessPoints.gsp]: org and not orgInstance?

File: app/grails-app/i18n/messages_de.properties [hunk 3]
```diff
@@ -801,6 +801,7 @@ org.nav.link=Meine Zugangskonfiguration
 org.nav.accessPoints = Zugangskonfigurationen
 org.nav.accessPoint.actions = Aktionen
 
+
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages_de.properties]: translations into English missing

**Comment by croqueGrec09** [comment]
cannot approve pull request, I have remarks

**Comment by mbeh** [comment]
Hallo Andreas, ich hab die besprochenen Punkte gefixt. Danke für das Review

**Pull request merged**